domains/cabbage.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Aaron's personal developer website",
+    "repo": "https://github.com/aaron-harvey",
+    "owner": {
+        "username": "cabbage",
+        "email": "1255651384@qq.com"
+    },
+    "record": {
+        "CNAME": "cabbage.com",
```

Comment:
This doesn't seem like a legitimate use of this service. If you want to hold this subdomain for the future, I'd suggest using a github pages host for your user.